content/release/ReleaseNotes373.md

```diff
@@ -0,0 +1,156 @@
+<!--
+title: "Contrast 3.7.3 - April 2020"
+description: "Contrast 3.7.3 April 2020"
+tags: "3.7.3 April Monthly Release News"
+-->
+
+# Monthly Release News - April 2020
+
+This page shows highlights from our all Contrast hosted, on-premises and agent releases over the past month.
+
+**3.7.3 on-premises release date: ** May 5, 2020
+
+**New and improved:**
+
+- In addition associating vulnerabilities, you can now also **associate both discovered and exercised routes** to build numbers, application versions, branches or repositories **using session metadata**. This means you can also query route information with a public API endpoint. With a single call to a public endpoint you can get detailed information on how much of an application has been exercised and where the critical vulnerabilities are.
+- You now have a choice to receive individual policy violation emails or to consolidate them into a single email. Find this option under **Organization Settings \&gt; Notifications**.
+- Your AppSec team can more easily assess library security risk and prioritize work with changes to surface CVE severity and make libraries easier to find. Select **Libraries** to see a **filterable list of libraries with visual display of CVE severity** for each one.
+
+**Important notes:**
+
+- To improve security, Contrast has updated our JRE version to Java 11 for both hosted and on-premises customers. This should not affect end users.
+
+**Bug fixes:**
+
+These significant bugs have been fixed in the past month:
+- SUP-1244 (TS-2697,TS-1494) 3.7.2 upgrade caused Contrast server and mysqld to run as the wrong user.
```

Comment:
Maybe I'm being overly pernickety, but I feel this should say:
"3.7.2 upgrade caused Contrast server and mysqld to _attempt to_ run as the wrong user"

Reply:
This is the kind of detail I'm hoping for, thank you.